hcl2template/types.packer_config.go

```diff
@@ -391,6 +473,16 @@ func (cfg *PackerConfig) GetBuilds(opts packer.GetBuildsOptions) ([]packer.Build
 			pcb.Provisioners = provisioners
 			pcb.PostProcessors = pps
```

Comment:
I think if we plan on not reusing these `provisioner` & `pps` we should not store them in `pcb` as it might make things confusing; also we probably need to shut them down because that means all plugins are started twice.

Reply:
( ehm if killing them sounds like an impossible hack then may be let's do that later )

Reply:
no way, this was super easy to do. I did it already in my local branch 👍